actionpack/lib/action_dispatch/http/resource_isolation_policy.rb

```diff
@@ -92,6 +92,10 @@ def call(env)
         if request.resource_isolation_policy &&
            Permissions.new(request, assets_prefix).forbidden?
 
+          if request.resource_isolation_policy.log_warning_on_fetch_metadata_failure
+            logger.warn "Fetch Metadata header didn't match request"
+          end
```

Comment:
I'm wondering if it wouldn't be better to just stick to one name for this concept - either "resource isolation policy" or "fetch metadata policy". Otherwise it may get confusing, wdyt?

Reply:
Yeah, I wasn't sure where the resource isolation term came from. I think for developers debugging this the term Fetch Metadata is more clear, since that's what the standard is called.